cypress/integration/abm-articulos/edit-article.js

```diff
@@ -0,0 +1,53 @@
+const { beforeEach } = require("mocha");
+
+describe ('Edit Article', function() {
+  beforeEach('Login', function() {
+    cy.visit('login');
+    cy.get('[formcontrolname=email]').type('santiago.lopez+1@wolox.com.ar');
+    cy.get('[formcontrolname=password]').type('Wolox1189!');
+    cy.get('.btn').click(); 
+    cy.get(':nth-child(4) > .nav-link').click(); 
+    cy.get(':nth-child(1) > .article-preview').click(); 
+    cy.contains("Edit Article").click()
```

Comment:
Please create a custom command to write common logic like login. https://docs.cypress.io/api/cypress-api/custom-commands.html